### DISCUSSION TITLE
Mass deletion of identical translations - Is this possible?

### DISCUSSION THREAD

**Discussion by luebbe** [WeblateOrg/weblate#6260] [Q&A] [answered]
I have received a set of .po files where for all untranslated strings the msgstr=msgid instead of msgstr="".

Obviously the weblate quality checks spot these cases.

Is there a mass edit function in weblate that allows me to delete all translations where `check:same`? I've only seen mass edit functions that allow me to edit the labels, tags or state of translations.

If not I have to solve it outside of weblate somehow.

**Comment by nijel** [comment, marked as answer]
Right now the bulk editing does not support changing content of the translation.

**Reply by luebbe** [reply]
Solved it outside of weblate. Thank you :)
